### ISSUE TITLE
Security: not auditable. is port to Google Sheets possible?

### ISSUE THREAD

**Issue by rinchik** [cgranier/wordlexcel#3]
Downloading and running scripted `*..xlsx` files is not advisable (contents is not auditable).

As an alternative you can share auditable scripting logic files with import instructions.

**Comment by cgranier** [comment]
Good point. I will look into this. I eventually want to lay out and explain all the logic and functions. I haven't tried it in Google Sheets -not sure if all the functions I used are supported.
